src/datasource.ts

```diff
@@ -383,6 +394,15 @@ export default class InstanaDatasource extends AbstractDatasource {
     return this.$q.resolve({data: {items: []}});
   }
 
+  getSloInformation(target, timeFilter: TimeFilter) {
+    // do not try to execute to big queries
+    if (this.isInvalidQueryInterval(timeFilter.windowSize, this.maxWindowSizeAnalyzeMetrics)) {
```

Comment:
do we need an own query interval or even none ?

Reply:
Copy-paste error..I think we don't need one because people might want to create weekly SLO reports and thus the query interval would an obstruction. Removed for now. Let me know what you think!